raiden/network/protocol.py

```diff
@@ -385,6 +385,15 @@ def peek(self, block=True, timeout=None):
     def __len__(self):
         return len(self._queue)
 
+    def snapshot(self):
```

Comment:
this could be renamed to copy, people have raised issues with the nomenclature of a snapshot method in a unrelated module.